packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1088,6 +1088,18 @@ base class PipelineOwner with DiagnosticableTreeMixin {
   bool _shouldMergeDirtyNodes = false;
   List<RenderObject> _nodesNeedingLayout = <RenderObject>[];
 
+  /// The [RenderObject]s which need to be laid out in the next [flushLayout] pass.
```

Comment:
nit: I'd say relayout boundaries since this is not a comprehensive list of dirty nodes.

Reply:
Done